piqueserver/config/config.json

```diff
@@ -24,6 +24,7 @@
         "Cheating isn't welcome. Griefing is frowned upon. Have fun!"
     ],
     "master" : false,
+    "ip_getter" : "https://services.buildandshoot.com/getip",
```

Comment:
hm, perhaps remove this from the default config? Few people will ever need to mess with it and it will just cause issues later if we change the defaults.